services/scabbard/libscabbard/src/client/reqwest/mod.rs

```diff
@@ -321,11 +321,11 @@ fn wait_for_batches(
         } else {
             let any_invalid_batches = batch_infos
                 .iter()
-                .any(|info| matches!(info.status, BatchStatus::Invalid(_)));
+                .any(|info| matches!(info.status, BatchStatus::Invalid(_) | BatchStatus::Unknown));
```

Comment:
The variable `any_invalid_batches` should match this new state - `any_invalid_or_unknown` as a suggestion.

Reply:
Good point! Updated